conda-forge.yml

```
@@ -1,10 +1,11 @@
-conda_forge_output_validation: true
-provider:
-  linux_aarch64: default
 build_platform:
   linux_aarch64: linux_64
```

Just as we cross-compile for `linux_aarch64`, think we should do the same with `linux_ppc64le`

```suggestion
  linux_aarch64: linux_64
  linux_ppc64le: linux_64
```

---

Also after making this change would [re-render]( https://conda-forge.org/docs/maintainer/updating_pkgs/#rerendering-feedstocks )